coretex/cli/commands/node.py

```diff
@@ -73,8 +73,9 @@ def stop() -> None:
 
 
 @click.command()
+@click.option("--auto", is_flag = True, help = "Configure node settings manually.")
 @onBeforeCommandExecute(node_module.initializeNodeConfiguration)
-def update() -> None:
+def update(auto: bool) -> None:
```

Comment:
Add "-y" and "-n" flags which define if a prompt will be automatically responded to with a yes or with a no